lua/entities/base_wire_entity.lua

```diff
@@ -9,6 +9,8 @@ ENT.AdminOnly = false
 
 ENT.IsWire = true
 
+ENT.lastUpdate = 0
```

Comment:
This var name is too general and may likely clash with a derived entity.

Reply:
im testing it now, i will rename.